lib/plugins/aws/deploy/compile/events/apiGateway/tests/authorizers.js

```diff
@@ -117,7 +117,7 @@ describe('#compileAuthorizers()', () => {
       expect(
         awsCompileApigEvents.serverless.service.provider.compiledCloudFormationTemplate
           .Resources.AuthorizerApiGatewayAuthorizer.Properties.AuthorizerResultTtlInSeconds
-      ).to.equal(400);
```

Comment:
Could you implement a separate test for testing that 0 gets properly set as well.